docs/development/mkdocs.zh.md

```diff
@@ -46,6 +46,28 @@
 - 编写文档的内容。您可以选择只写英文文档或中文文档；您不必用两种语言编写文档；当然如果您希望展示您中英双语的实力的话，我们建议您两种语言的文档同时编写，一并提交。
 - 更新`/mkdocs.yml`文件，更新`nav:`部分。这是整个文档网站的目录。
```

Comment:
Please synchronize with the English version here 
<img width="1166" alt="image" src="https://user-images.githubusercontent.com/16207029/175493372-379847ec-2f35-4467-bc19-778da9d4a8b6.png">